Assets/Scripts/Player.cs

```diff
@@ -0,0 +1,62 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+
+public class Player : MonoBehaviour {
+
+    private Camera camera;
+
+    void Start() {
```

Comment:
uma coisa que me deixa muito triste, mas aparentemente é o padrao do c#, iniciar todas as aberturas de chaves na linha seguinte :/
```suggestion
    void Start() 
    {
    }
```

ai isso vale, pra metodos, ifs e afins

Reply:
PepeSadge. Vou alterar aqui.